spot-ml/src/main/scala/org/apache/spot/dns/DNSWordCreation.scala

```diff
@@ -104,11 +93,11 @@ class DNSWordCreation(frameLengthCuts: Array[Double],
         extractDomainInfo(queryName, topDomainsBC, userDomain)
 
       Seq(topDomain,
-        Quantiles.bin(frameLength.toDouble, frameLengthCuts),
-        Quantiles.bin(unixTimeStamp.toDouble, timeCuts),
-        Quantiles.bin(subdomainLength.toDouble, subdomainLengthCuts),
-        Quantiles.bin(subdomainEntropy, entropyCuts),
-        Quantiles.bin(numPeriods.toDouble, numberPeriodsCuts),
+        MathUtils.logBaseXInt(frameLength.toDouble, 2),
```

Comment:
Good, in the future we could assign a base other than 2 to play around and see if we are able to improve performance.